src/app/volunteer/register/[...slug]/style.css.ts

```diff
@@ -0,0 +1,24 @@
+import { style } from '@vanilla-extract/css';
+
+export const wrapper = style({
+  boxSizing: 'border-box',
+  padding: '40px 0px'
+});
+
+export const TitleSection = style({
```

Comment:
lower CamelCase로 통일성있게 사용하면 좋을 것 같아요

```suggestion
export const titleSection = style({
```

Reply:
앗 꼼꼼한 리뷰 감사합니다!!